src/gsi/setupt.f90

```diff
@@ -1375,7 +1375,7 @@ subroutine init_vars_
         varname='th2m'
         call gsi_bundlegetpointer(gsi_metguess_bundle(1),trim(varname),rank2,istatus)
         if (istatus==0) then
-            if(allocated(ges_z))then
+            if(allocated(ges_th2m))then
```

Comment:
On line 1378, please replace ges_th2m with ges_th2.  There is no variable, ges_th2m, so the code fails to compile.